src/pykeen/losses.py

```diff
@@ -530,6 +536,12 @@ def forward(
             neg_scores - pos_scores + self.margin,
         ))
 
+    def get_config(self) -> MutableMapping[str, Any]:  # noqa: D102
+        config = super().get_config()
+        config["margin"] = self.margin
+        config["margin_activation"] = margin_activation_resolver.normalize_inst(self.margin_activation)
```

Comment:
what if the activation has kwargs? should this be recursive?

Reply:
Yes, this should also consider `kwargs` :thinking: Since activations usually are PyTorch objects, this is may be problematic  - so maybe we need the second alternative described above, where the resolver gets the `get_config` method?